rust/relay/ebpf-turn-router/src/main.rs

```diff
@@ -121,6 +120,7 @@ fn try_handle_turn(ctx: &XdpContext) -> Result<u32, Error> {
     };
 
     // If we get to here, we modified the packet and need to send it back out again.
+    debug!(ctx, "Successfully processed packet, transmitting with XDP_TX");
```

Comment:
Do we need all of these to be on debug? I think a lot of them would be better at `trace`, othewise debug is very spammy.